guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>The resolution of this screenshot doesn't look very good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The initial size of the image was big. And it looked odd to me,, So I have decreased the size which has caused this image to look blurry. Please have a look at the same image in <a href="https://github.com/Esri/arcgis-python-api/blob/3adbc83829584f9b6aa0197cd8272240ab0ff0a1/guide/14-deep-learning/How_Instance_segmentation_MaskRCNN_works.ipynb" target="_blank">https://github.com/Esri/arcgis-python-api/blob/3adbc83829584f9b6aa0197cd8272240ab0ff0a1/guide/14-deep-learning/How_Instance_segmentation_MaskRCNN_works.ipynb</a>. If this size looks fine to you, I can replace it back.</p>